### PR TITLE
Convert from using id to slug for admission

### PR DESCRIPTION
**Pull request by norbye**
Rename all api endpoints from using id reference to using slugs.

Set to merge into `cleanup-models` to make the diff more informative and to prevent conflicts down the road. Will change to master after #1011 is merged and rebase potential changes.

Resolves ABA-369

### CODE REVIEW

File: admissions/admissions/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .test_api import *
```
Comment (LudvigHz) [inline admissions/admissions/tests/__init__.py:1]: This should not be necessary. Pls remove

File: frontend/src/routes/ManageAdmissions/CreateAdmission.tsx [hunk 2]
```diff
@@ -131,6 +131,19 @@ const CreateAdmission: React.FC = () => {
...
+            Opptaket vil ligge under opptak.abakus.no/[slug]/
+          </InputDescription>
+          <Input
+            name="slug"
+            value={formik.values.slug}
+            onChange={formik.handleChange}
+            placeholder="f. eks. komiteer/revy/backup"
+            disabled={!isNew}
+          />
+        </InputWrapper>
```
Comment (LudvigHz) [inline frontend/src/routes/ManageAdmissions/CreateAdmission.tsx:146]: I'm assuming validation errors are handled by the backend here?

**Review by LudvigHz — APPROVED** [review]
Awesome!

File: frontend/src/routes/ManageAdmissions/CreateAdmission.tsx (hunk 2 above)
Comment (norbye) [reply frontend/src/routes/ManageAdmissions/CreateAdmission.tsx:146]: Yas (at least the basics for a slug, haven't added more than that), had some problems with the way the frontend does requests regarding fetching error data, so I'm awaiting refactoring that code to display the errors properly in the frontend.

Although I have completely forgotten to add frontend validation at all to that form - so I'll write that up to take a look at improving it later(:

**Pull request merged**